lecture-demo/lectureDemo.tex

```diff
@@ -0,0 +1,78 @@
+\documentclass[
+	aspectratio=169, % default is 43
+	8pt, % font size, default is 11pt
+]{beamer}
+
+\usepackage{../beamerthemeuulm} % use the inofficial uulm beamer theme
+
+\title{Beamer Lecture Test}
```

Comment:
would be good to use a caption here such that it is clear whether this is the lecture series or a single 90-minute lecture